docs/index.md

```diff
@@ -20,6 +20,17 @@ The HCP provider supports authentication via a Client ID and a Client Secret. Th
 
 ## Example Usage
 
+Everything in HashiCorp Cloud Platform (HCP) starts with the HashiCorp Virtual Network (HVN).
```

Comment:
Should we change the heading from `Example Usage` to something else like `Getting Started`? When I read this section within the context of the `Example Usage` heading, some of the explanation and instruction seemed slightly out of place.